src/GojsDiagram.tsx

```diff
@@ -79,7 +80,8 @@ class GojsDiagram<N extends BaseNodeModel, L extends LinkModel> extends React.Pu
             linkFromPortIdProperty: this.props.linkFromPortIdProperty || '',
             linkToPortIdProperty: this.props.linkToPortIdProperty || '',
             nodeDataArray: [...this.props.model.nodeDataArray],
-            linkDataArray: [...this.props.model.linkDataArray]
+            linkDataArray: [...this.props.model.linkDataArray],
+            nodeCategoryProperty: this.props.nodeCategoryProperty || 'category'
```

Comment:
is it possible to add a test stating that the default value of the property is 'category' otherwise it is what was set in `this.props.nodeCategoryProperty`  ?

Reply:
I will add the test to the current commit.

Reply:
@soshimozi could you add a unit test ?
We would like to merge and close this pull request quickly.

Reply:
@nicolaserny So sorry, had some personal issues that prevented me from getting to this sooner.  I created a UT for the default category setting.

Reply:
@soshimozi Thanks :) We will merge your PR and create a new release